packages/migrations/src/actions/2024.11.07T00.00.00.create-preflight-scripts.ts

```diff
@@ -0,0 +1,23 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2024.11.07T00.00.00.create-preflight-scripts.sql',
```

Comment:
```
  name: '2024.11.07T00.00.00.create-preflight-scripts.sql',
```

TODO @dimaMachina use correct file extension

Reply:
you are right, let's use `const filename = path.parse(import.meta.url).base` in all migrations? and `await import('./actions/...YOUR_MIGRATION_FILE'),` like you did for supertokens?